client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -54,10 +54,10 @@ const ThemesRelatedCard = React.createClass( {
 		return (
 			<div>
 				<SectionHeader label={ i18n.translate( 'You might also like' ) } />
-				<ul className="themes__sheet-related-themes">
+				<ul className="themes-related-card__">
```

Comment:
>cc @aduth as we should probably not allow trailing `__`.

From a linting point-of-view, I suppose it's enough that we ensure there's at least one more character after the `__` ?